root/static/scripts/work.js

```diff
@@ -268,6 +267,8 @@ function removeLanguage(i) {
 const LANG_MUL_ID = 284;
 const LANG_ZXX_ID = 486;
 
+const getSelectField = field => field;
```

Comment:
Why not just have this directly in the attribute?

Reply:
It's usually a bad idea to do that because the component could be using `React.memo`. The component would never be memoized, because you're passing a different function every time. (On HTML elements this doesn't matter.)

This function also doesn't depend on any value inside the scope of `renderWorkLanguages`, so there's no technical need to redefine it either.